build_tools/benchmarks/post_benchmarks_as_pr_comment.py

```diff
@@ -148,15 +148,18 @@ def get_benchmark_result_markdown(benchmark_files: Sequence[str],
       base_commit = md.link(base_commit,
                             f"{GITHUB_IREE_REPO_PREFIX}/commit/{base_commit}")
 
-      if len(base_benchmarks) == 0:
-        commit_info = (f"@ commit {pr_commit} (no previous benchmark results to"
-                       f" compare against since {base_commit})")
-        continue
-
       # Update the aggregate benchmarks with base numbers.
+      any_matched_bench = False
       for bench in base_benchmarks:
         if bench in all_benchmarks:
+          any_matched_bench = True
```

Comment:
Don't we want to get one with all matched benchmarks if possible? Also this really should be querying based on the merge base of the PR (and we should be using a database)...

Reply:
Changed to check all matched benchmarks. I'm okay with both ways. Checking all benchmarks will makes it not be able to find the base when the PR adds some new benchmarks. But I think it is fine in that case.

Reply:
Ah that's actually a good point. We probably do want comparison points if someone is adding a new benchmark. I think either way is fine. Really we need to upgrade the way this is done overall, so I'll avoid spending too much time bikeshedding.